seisflows/solver/specfem3d.py

```diff
@@ -252,3 +253,99 @@ def combine_vol_data_vtk(self, input_path, output_path, hi_res=False,
             stdout = f"{self._exc2log(exc)}_{name}.log"
             self._run_binary(executable=exc, stdout=stdout, with_mpi=False)
 
+    def make_output_vtk_files(self, input_path, output_path=None, 
+                              parameters=None, hi_res=False, tag=None, 
+                              kernel=False):
+        """
+        A warpper on `combine_vol_data_vtk()` that automatically tries to 
+        generate .vtk files using the SPECFEM binary xcombine_vol_data_vtk, 
+        and rename the output files to not be so generic. Files will be stored
+        in the `output_path` directory, and will be named based on the `tag` 
+        unless overwritten by the User.
```

Comment:
```suggestion
        A wrapper for `combine_vol_data_vtk()` that automatically tries to 
        generate .vtk files using the SPECFEM binary xcombine_vol_data_vtk, 
        and rename the output files to make them less generic. Files will be stored
        in the `output_path` directory, and will be named based on the `tag` 
        unless overwritten by the user.
```

Reply:
Hi @VascoSch92, I appreciate you catching some typos, I can fix those before I PR, though I admit I am a bit confused on the need for such a minor code review?